content/en/blog/multi-stage-docker-build/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Multi Stage Docker Build 🐳 "
+description: "Hsi"
+excerpt: "Optimize Docker images with multi-stage builds to reduce size,"
+date: 2024-09-26T21:22:41+00:00 
+lastmod: 2024-09-26T21:22:41+00:00 
+draft: false
+weight: 90
+images: [multi-stage-image.jpg]
+categories: ["News"]
```

Comment:
Can you change the categories to the one related to the article.

Reply:
I have updated the categories